eng/Publishing.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <!-- The one use of ArtifactsDir in Publish.proj adds an additional slash, confusing itself. -->
+  <PropertyGroup Condition=" HasTrailingSlash('$(ArtifactsDir)') ">
+    <ArtifactsDir>$(ArtifactsDir.Substring(0, $([MSBuild]::Subtract($(ArtifactsDir.Length), 1))))</ArtifactsDir>
+  </PropertyGroup>
+
+  <!-- Include our "loose" PDBs when publishing. -->
+  <ItemGroup>
+    <FilesToPublishToSymbolServer Include="$(SymbolsOutputPath)**\*.pdb" />
```

Comment:
This line did not add the expected Microsoft.Web.Xdt.Extensions.pdb file to PDBArtifacts. Nothing logged about uploading that file in https://dev.azure.com/dnceng/internal/_build/results?buildId=334866&view=logs&s=6884a131-87da-5381-61f3-d7acc3b91d76&j=1b89928a-2219-5ef9-602f-f95beb3da4dc and the build produced no PDBArtifacts.

@riarenas @JohnTortugo @mmitche should we be setting `$(UsingToolSymbolUploader)` as well or instead of adding this item? Or, is placing the file in the PDBArtifacts directory our only option?